pkg/apis/flowcontrol/types.go

```diff
@@ -318,6 +331,18 @@ type FlowSchemaCondition struct {
 // FlowSchemaConditionType is a valid value for FlowSchemaStatusCondition.Type
 type FlowSchemaConditionType string
 
+// EvaluatedPriorityLevelConfigurationReference is a report on whether
+// a PriorityLevelConfigurationReference refers to something that
+// exists
+type EvaluatedPriorityLevelConfigurationReference struct {
+	// `reference` is the reference being reported upon
+	Reference PriorityLevelConfigurationReference
+
+	// `exists` reports whether the referenced priority level
+	// configuration object exists
+	Exists bool
```

Comment:
status fields for referential integrity status is also fairly unusual...

Reply:
> status fields for referential integrity status is also fairly unusual...

While  I may choose to frame this as a condition, a referential checks seems inbounds in cases of two cluster-scoped resources managed by cluster-admins.  The question of "is it valid" answered in status seems reasonable.

Reply:
We have "is it working yet" conditions for CRDs. I think this is extremely similar. I might phrase it as a single e.g. "operational" condition, with differing reasons depending on why it's not operational.